src/services/categories.services.ts

```diff
@@ -1,17 +1,21 @@
 import {api} from "@/boot/axios";
```

Comment:
```suggestion
import { api } from "@/boot/axios";
```

Anh cũng hay gặp lỗi này khi code mà quên config IDE, cơ bản là mình cần thêm 1 space vào như trên và mình cần config cái IDE để nó làm cái này tự động

![image](https://user-images.githubusercontent.com/10450325/132133467-55c5d75e-4553-4407-a5c0-e663b6fb05c1.png)